### DISCUSSION TITLE
Suppress warnings from submodules

### DISCUSSION THREAD

**Discussion by J-Gusmao** [pycaret/pycaret#3720] [Q&A]
Hello.
I am using the library to select the best model for a project, and I am limiting the tested models.
The thing is that, even though I remove LightGBM from the models in "compare_models", it keeps printing its warnings in the console.
Is there a way to suppress them, even if I was using the module, as I want to keep my console clean?

Example of the INFO and WARNINGS:
<img width="377" alt="image" src="https://github.com/pycaret/pycaret/assets/121870711/3a065519-6933-4b09-9683-966776b2a3d1">

Again, I'm not trying to train or test the model, this appears after I run the "setup" for Regression Experiment

**Comment by celestinoxp** [comment]
Issue about LightGBM 4.0 https://github.com/pycaret/pycaret/issues/3660

**Reply by J-Gusmao** [reply]
I tried the solution in the post and it did not work too.
